First project

tingamapuro04 opened the pull request:
added 
1.index.html
2. styles.css
3. test.md
4. gitignore
5. linters.yml

tingamapuro04 committed:
linters.yml added

tingamapuro04 committed:
added files

tingamapuro04 committed:
heading 1

tingamapuro04 committed:
linters changes

tingamapuro04 committed:
changes

Gambit142 commented on .gitignore:
```diff
@@ -0,0 +1 @@
+test.md
```
- [ ] Kindly include `node_modules` in your `.gitignore` file. Including `node_modules` in `.gitignore` files is an industry best practice because the folder contains a lot of modules that you wouldn't want to push to a hosting site like Github. Kindly implement this change. 👍

Gambit142 reviewed (changes requested):
Hi @tingamapuro04 👋, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Highlights

- Linters checks passed 💯

## Required Changes ♻️

- Kindly make your README.md file professional. You can use this Microverse template https://github.com/microverseinc/readme-template/blob/master/README.md as a guide to making all your README.md files during your learning period in Microverse. 👍 
- Kindly make your pull request summary a little descriptive. It should explain what the current branch you worked on introduced to the project. 👍 
- _Kindly check the comments under the review for further required changes._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

- Since it would be tedious to change your commit messages, I would like to advise you to read this article https://www.freecodecamp.org/news/how-to-write-better-git-commit-messages/ and use it as a guide to writing better commit messages in the future projects. 👍 

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._
```

tingamapuro04 committed:
Added node_modules folder in the .gitignore

tingamapuro04 committed:
Added specifications on the README.md

tingamapuro04 commented:
I am working on my feature branch where I have
1. index.html
2. styles.css
3. .gitignore
4. linters.yml
5. test.md
I have added more details on the README.md and also I have added node_modules folder to the .gitignore file

OLIPLICHE commented on README.md:
```diff
@@ -1,2 +1,19 @@
 # Greetings-Microverse
```
## Optional
- Please I suggest that you update your Readme and make it looks like the one in the template provided to you by Microverse. Open this link to read more: [README template](https://github.com/microverseinc/readme-template/blob/master/README.md)

OLIPLICHE reviewed (changes requested):
Hi @tingamapuro04, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!
<FEEL FREE TO CHANGE THIS MESSAGE TO YOUR PERSONAL WORDS>

## Highlights
- Linter is green (the last commit have a green check mark (✅)) 👏
- README file is descriptive 👏
- Commit messages are meaningful 👏 👏
- Github flow is correctly used (Pull Request is made from the feature branch into the main branch) 👏
- Pull Request have a descriptive title telling you what changes are inside 👏
- Pull Request have a short summary describing introduced changes in more detail👏
- `node_modules` and `test.md` folders are in the `.gitignore` file.👏

## Required Changes ♻️

- [ ] You did well when you add `test.md` inside the `gitignore` file. Congratulations. Please, detele the `test.md` file in your GitHub repository because it is the requirement of the project.
-  Please delete it.

![git](https://user-images.githubusercontent.com/63972670/165449840-ff2a7087-1762-4046-ac1d-53d5c6df61e4.PNG)





## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._



Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

tingamapuro04 committed:
Deleted test.md

tingamapuro04 commented:
I am working on the feature branch and I have deleted the test.md file from the repository after adding it the .gitignore

vic778 commented on README.md:
```diff
@@ -1,2 +1,19 @@
 # Greetings-Microverse
-Hello Microverse
+Microverse first project.
+
+## Version
+Version 1.0.0
+
+## Contributors
+
+**Author**
+Name : Adoyo Alphonce;
+Email : apondimama5@gmail.com;
+Github : https://github.com/tingamapuro04
+
+
+## Description
+This is a small project generally looking at github workflow where i was setting a linters.yml file, adding a .gitignore and then created a heading and added a color to it.
+
+## Copyright
+© Adoyo A Mobutu
```
- [ ] [OPTIONAL] kindly you have to make change requested by the previous review. you're almost there but follow this [microverse Readme](https://github.com/microverseinc/readme-template/blob/master/README.md)

vic778 reviewed (approved):
Hi @tingamapuro04 , 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

![Charlie Brown](https://media.giphy.com/media/ZdO4NenDbsQNwUwKDB/giphy.gif)

## Highlights

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._
